plugins/chat/assets/javascripts/discourse/components/chat-channel-members-view.js

```diff
@@ -1,24 +1,20 @@
-import { isEmpty } from "@ember/utils";
 import { INPUT_DELAY } from "discourse-common/config/environment";
 import Component from "@ember/component";
 import { action } from "@ember/object";
 import { schedule } from "@ember/runloop";
-import ChatApi from "discourse/plugins/chat/discourse/lib/chat-api";
 import discourseDebounce from "discourse-common/lib/debounce";
-
-const LIMIT = 50;
+import { inject as service } from "@ember/service";
 
 export default class ChatChannelMembersView extends Component {
+  @service chatApi;
+
   tagName = "";
   channel = null;
-  members = null;
   isSearchFocused = false;
-  isFetchingMembers = false;
   onlineUsers = null;
-  offset = 0;
   filter = null;
   inputSelector = "channel-members-view__search-input";
-  canLoadMore = true;
+  membershipsCollection = null;
```

Comment:
Nitpick: I'm not sure that `membershipsCollection` is better than `members` here, seems needlessly long. (`channelMembers` would be fine too, to match the component name.)